src/firefly/js/tables/TableInfoUtil.js

```diff
@@ -97,10 +98,12 @@ export function getCenterColumns(table) {
  *
  */
 function guessDefColumns(table) {
-    const {columns}= table.tableData;
-    const colList= columns.map( (c) => c.name.toLowerCase());
-    if (colList.includes('ra') && colList.includes('dec')) return 'ra;dec;EQ_J2000';
-    if (colList.includes('lon') && colList.includes('lat')) return 'lon;lat;EQ_J2000';
-    return 'ra;dec;EQ_J2000';
+    // ex. return 'ra;dec;EQ_J2000' or 'lon;lat;EQ_J2000'
+    const guess = (lon, lat) => {
+        const lonCol = getColumn(table, lon, true);
+        const latCol = getColumn(table, lat, true);
+        if (lonCol && latCol) return `${lonCol.name};${latCol.name};EQ_J2000`;
+    };
+    return guess('ra','dec') || guess('lon', 'lat');
```

Comment:
I guess 'ra','dec', 'lon'&'lat' guess applied to most cases, right? or could we make the 'guess' configurable based on the mission? just a thought.

Reply:
I think, this is mainly for uploaded tables, as the mission-related catalogs have "CatalogCoordColumns" set in meta. I did not change this method, only made it to work with case independent column names, as was the original intention.